File: Python/Stacks&Queue/Stack.py
class Stack:
    
    def __init__(self):
        self.items = []
    
    def push(self,item):
        self.items.append(item)
    
    def peek(self):
        return self.items[-1]
    
    def pop(self):
        return self.items.pop()

    def __len__(self):
        return len(self.items)


if __name__=="__main__":
    s = Stack()
    s.push(5)
    s.push(15)
    s.push(55)
    s.push(53)
    
    for i in range(len(s)):
        print(s.pop())

File: C++/Stacks_Queue/Stack.cpp
#include <iostream>
#include <vector>

class Stack
{
public:
    Stack();
    void push(int value);
    int pop();
    int size();
private:
    std::vector<int> items;
};



int main()
{
    Stack q;
    q.push(1);
    q.push(3);
    q.push(5);
    q.push(7);
    q.push(2);
    for (int i{};i<5;i++)
    {
        std::cout<<q.pop()<<std::endl;
    }
    return 0;
}


Stack::Stack()
{
    items = std::vector<int>();
}

void Stack::push(int value)
{
    this->items.push_back(value);
}

int Stack::pop()
{
    int n = items[items.size()-1];
    this->items.pop_back();
    return n;
}

int Stack::size()
{
    return this->items.size();
}